scripts/GenerateUnicodeBlock.sc

```diff
@@ -0,0 +1,415 @@
+// Run with $ amm scripts/GenerateUnicodeBlock.sc
+//   Replace code in ./javalanglib/src/main/scala/java/lang/Character.scala
```

Comment:
Please see comments in Character.scala, which I will review next.  IMO, the software engineering would be improved if this script printed out some, actually quite a bit of audit
trail information:  Who ran it, when, which version of Unicode used as input files.